### ISSUE TITLE
Add Scratch usernames beside Minecraft usernames

### ISSUE THREAD

**Issue by liam4** [quatcraft/quatcraft.github.io#1]
> QuatCraft is a whitelisted vanilla survival Minecraft server for Minecraft 1.9.2 - by and for Scratchers.

Doesn't it just kind of make sense to add the Scratch usernames beside the Minecraft usernames?

**Comment by quat1024** [comment]
Well, yeah, but I'd have to go and collect them all.

Could maybe do it on hover or something, or add links to Scratch profiles.

**Comment by quat1024** [comment]
- [ ] load list of user names from nodecraft (<-- I hope I can write scripts on nodecraft, else do a bukkit plugin/leech off of Dynmap's web server)
- [ ] clicking on someone's name should open a small modal
- [ ] with their skin render from craftatar (<-- should I cache these? that's another script)
- [ ] and a link to their scratch profile
- [ ] and their name color and rank (ranks I can do manually, I don't have to load them from pex because I almost never add mods)

a lot to take on, but it will make the site awesome :package:

**Comment by scratchyone** [comment]
I have an idea: Maybe let users enter their scratch username on the site, and then you just have to verify.

**Comment by quat1024** [comment]
Good idea but I can't do any server-side coding. I already have a list of everyone's Scratch usernames anyways (took some googling but I did it)

**Comment by scratchyone** [comment]
Ok.

That confused me for a second. Github's notification called you by name, so I had no idea who you were :stuck_out_tongue:

**Comment by quat1024** [comment]
Sorry about that, I've just changed my name to something more people online call me by.

My real name isn't that interesting anyways ^^

**Comment by scratchyone** [comment]
That's better :P

I wish github called people by usernames and not names.

On Friday, May 13, 2016, quaternary <notifications@github.com> wrote:

> Sorry about that, I've just changed my name to something more people
> online call me by.
>
> My real name isn't that interesting anyways ^^
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/quatcraft/quatcraft.github.io/issues/1#issuecomment-219139041>
>


--
